src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,12 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        boolean result = ((a && b && !c && !d)
+                || (a && c && !b && !d)
+                || (a && d && !c && !b)
+                || (!a && b && c && !d)
+                || (!a && b && !c && d)
+                || (!a && !b && c && d)) ? true : false;
```

Comment:
try to use ^ (XOR) here, that might make your solution shorter

Reply:
done